CONTRIBUTING.md

```diff
@@ -231,6 +231,24 @@ Writing separate tests might be much better than making a single test that check
 
 Also, we should use driver methods as much as possible instead of testing commands directly via `RunCommand`.
 
+#### Integration tests naming guidelines
+
+1. Test files should be named after the command they test.
+   For example, `find_test.go` for testing the find command.
+   Test files belonging to the same commands group are organized into folders named after this group,
```

Comment:
This is a question to everyone, do we want a test file for each operator such as `find_bitwiseanyset_compat_test.go` or are we happy grouping them in a same file with _similar_ operators like now? Not sure how we've grouped them now but it makes sense.

Reply:
To be honest, I don't like the current approach of having everything in the same file because it's hard to navigate through the files that have 1000+ lines of code. If the files for different operators are separate, it's easier to go through them, scroll through the tests for a particular operator, etc.

Reply:
Yeah agree. If we all agree let's document that too :hugs:

Reply:
Yeah, `find`, `update,` `aggregate` will be huge without additional level of splitting